api/python/quilt3/telemetry.py

```diff
@@ -0,0 +1,118 @@
+import functools
+from concurrent.futures import ThreadPoolExecutor, wait
+from requests_futures.sessions import FuturesSession
+from threading import Lock
+import uuid
+import atexit
+import time
+
+TELEMETRY_URL = "https://gha7zraiy7.execute-api.us-east-1.amazonaws.com/Stage/metrics/installer"
```

Comment:
Quick suggestion: use a DNS name for this (so we don't have to update clients when we redeploy the service).

Reply:
It's a test API for now, but it will go to quiltdata.cloud/metrics when this is finalized

Reply:
Definitely need a DNS name for this... But does API Gateway even support it? Or would we make it part of the registry?

Reply:
API Gateway supports custom domain names. The Telemetry API in the prod account is already set up to be called via quiltdata.cloud/metrics.